package.json

```diff
@@ -10,7 +10,7 @@
     "vite:dev": "vite",
     "vite:build": "vue-tsc --noEmit && vite build",
     "vite:preview": "vite preview",
-    "tauri:dev": "yarn tauri dev",
+    "tauri:dev": "tauri dev",
```

Comment:
`yarn tauri dev` will make it re-install all dependencies before each launch, that's why `yarn` was removed